script/bnb/verify_ynclisbnbk.s.sol

```diff
@@ -52,6 +52,7 @@ contract VerifyYnclisBNBkStrategy is BaseVerifyScript {
 
         _verifyApprovalRule(vault, contracts.WBNB(), contracts.STAKER_GATEWAY());
         _verifyStakingRule(vault, contracts.STAKER_GATEWAY(), contracts.WBNB());
+        _verifyUnstakingRule(vault, contracts.STAKER_GATEWAY(), contracts.WBNB());
```

Comment:
@danoctavian do we need staking and unstaking rules for the clisBNB vault? Amd if so should it be `stakeClisBNB` and unstake OR `stake` and unstake for wbnb?

Reply:
for clisBNB vault we want the stakeClisBNB unstakeClisBNB ones

Reply:
fixed in [e4936dd](https://github.com/yieldnest/yieldnest-kernel-lrt/pull/33/commits/e4936dd8f13462fdf0e237ffa07ddb990e11d9f4)